src/test/functional/pages/BuzzPage.ts

```diff
@@ -10,22 +10,29 @@ export class BuzzPage extends BasePage {
     readonly photoImg: Locator
     readonly noPostParagraph: Locator
     readonly buzzPageButton: Locator
+    readonly editPostParagraph: Locator
+    readonly textFieldInPostEdit: Locator
+    readonly removePictureButton: Locator
+    readonly confirmEditedPost: Locator
   
     constructor(page: Page) {
       super(page)
-      this.sharePhotosButton = page.locator('button:has-text("Share Photos")');
+      this.sharePhotosButton = page.getByText("Share Photos");
```

Comment:
Better to use roles for buttons:
page.getByRole("button", { name: "Share Photos" })